### DISCUSSION TITLE
dolphinscheduler 3.1.9版本去除zookeeper

### DISCUSSION THREAD

**Discussion by Bosshzh** [apache/dolphinscheduler#15643] [Q&A]
dolphinscheduler版本：3.1.9
想问下部署dolphinscheduler 3.1.9集群版本，能否去除掉zookeeper作为注册中心，而是使用mysql或者其它组件来实现 Master 节点的选举、Worker 节点的注册、心跳检测和worker任务分发等功能
如何要实现上诉功能，是需要修改源码重新编译吗，还是有现成的部署包呢？

I would like to ask if I can remove Zookeeper as the registration center when deploying DolphinScheduler 3.1.9 cluster version, and instead use MySQL or other components to implement functions such as Master node election, Worker node registration, heartbeat detection, and worker task distribution. If you want to achieve the above functionality, do you need to modify the source code and recompile it, or are there ready-made deployment packages available?

**Comment by CHINAJAVABoy** [comment]
你都知道集群需要注册中心，不用zk你就看看市面有哪些工具替代咯，况且zk不仅仅用作注册中心还用它的分布式锁

**Reply by zhangyulei000** [reply]
我standalone部署为什么也要启动zookeeper呢？

**Comment by CHINAJAVABoy** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
